packages/hathor-rpc-handler/src/rpcMethods/getUtxos.ts

```diff
@@ -13,11 +14,27 @@ import {
   UtxoDetails,
   RpcResponseTypes,
   RpcResponse,
+  RpcMethods,
 } from '../types';
-import { PromptRejectedError } from '../errors';
+import { PromptRejectedError, InvalidParamsError } from '../errors';
 import { TriggerTypes } from '../types';
 import { validateNetwork } from '../helpers';
 
+const getUtxosSchema = z.object({
+  method: z.literal(RpcMethods.GetUtxos),
+  params: z.object({
+    network: z.string(),
```

Comment:
In other methods you used `z.string().ming(1)` for network.

Why not in this one?

Reply:
Done! Thanks